### ISSUE TITLE
Mulebots can create teleporting human-chair hybrids

### ISSUE THREAD

**Issue by Madjura** [tgstation/-tg-station#11099]
Server revision compiled on: 2015-08-07
951ddb3ff412d055eb4dcac5973e2db0c24b1563 

How to:
Buckle someone to a roller chair (I assume roller beds would also work or the janicart)
Hack a Mulebot to make it accept anything, not just crates (load platform clunks wire)
Drag the chair on the Mulebot
Drag the still buckled person away to make the chair teleport on their head
Enjoy your terrifiying human-chair hybrid
You can drag the same chair on another Mulebot to have the chair float higher on the person
You can then ask someone to push the human-chair hybrid on your sprite (rest and have someone push it or move you under it) and then buckle yourself to the chair to become a human-chair hybrid and propel yourself with a fire extinguisher (can't walk due to being buckled)

If someone unloads the Mulebot then human-chair hybrid teleports away to where the Mulebot unloads

I present to you Honky the Human-Chair Clown

![ss13 human chair](https://cloud.githubusercontent.com/assets/4841238/9149730/8b214a6e-3db7-11e5-960b-41f22af01ec3.png)
![ss13 human chair2](https://cloud.githubusercontent.com/assets/4841238/9149731/8e7cf668-3db7-11e5-913d-caa1c22d1091.png)

**Comment by WJohn** [comment]
Holy shit that's an edge case if I've ever seen one. How do you people FIND this shit?

**Comment by Madjura** [comment]
I wanted to direct the corpse on top of the Mulebot to the bar or somewhere and then stop the Mulebot and declare it king of the bar
People can't steal stuff off a locked Mulebot (unless they whack it I think) but only the chair moved on the Mulebot and not the corpse, but the corpse was standing up. When I bumped into it it created the human-chair hybrid and I thought "What happens if I push the chair on the other Mulebot"
Tl;dr Clown doing clown things

**Comment by Dorsisdwarf** [comment]
AYY LMAO is for abductor issues

**Comment by Shapsy** [comment]
Holy shit I was the RD you showed this to in game earlier, I was really, really goddamn confused. I hope the Captain gave you a medal for your research

**Comment by phil235** [comment]
That's because the mulebot loading code probably doesn't call Move() but directly set the loc of the thing to be the mulebot or something.

**Comment by Miauw62** [comment]
bug of the year 2015

**Comment by Core0verload** [comment]
There is a ton of edge cases with mulebot loading, but this one is fucking hilarious.

**Comment by bgobandit** [comment]
they have eyes

THEY
HAVE 
EYES

**Comment by Cebasiz** [comment]
clap clap clap clap
